mltsp/tests/test_build_model.py

```diff
@@ -25,57 +25,147 @@ def test_csv_parser():
-                              "test_build_model_RF.pkl"))
+                              "NEW_MODEL_KEY.pkl"))
     assert hasattr(model, "predict_proba")
-    os.remove(pjoin(cfg.MODELS_FOLDER, "test_build_model_RF.pkl"))
+    os.remove(pjoin(cfg.MODELS_FOLDER, "NEW_MODEL_KEY.pkl"))
     assert not os.path.exists(pjoin(cfg.MODELS_FOLDER,
-                                    "test_build_model_RF.pkl"))
+                                    "NEW_MODEL_KEY.pkl"))
 
 
 def test_read_features_data_from_disk():
     """Test read features data from disk"""
-    for suffix in ["features.csv", "classes.npy"]:
+    for suffix in ["features.csv", "targets.npy"]:
         shutil.copy(
             pjoin(DATA_PATH, "test_%s" % suffix),
             pjoin(cfg.FEATURES_FOLDER, "TEST001_%s" % suffix))
     data_dict = ctt.read_features_data_from_disk("TEST001")
-    npt.assert_array_equal(data_dict["classes"], ["Mira", "Herbig_AEBE",
-                                                  "Beta_Lyrae"])
-    for fname in ["TEST001_features.csv", "TEST001_classes.npy"]:
+    npt.assert_array_equal(data_dict["targets"], ['Mira', 'Herbig_AEBE',
+                                                  'Beta_Lyrae', 'Mira',
+                                                  'RR_Lyrae', 'Mira'])
+    for fname in ["TEST001_features.csv", "TEST001_targets.npy"]:
         os.remove(pjoin(cfg.FEATURES_FOLDER, fname))
 
 
-def test_build_model():
-    """Test main model building method"""
-    shutil.copy(pjoin(DATA_PATH, "test_classes.npy"),
-                pjoin(cfg.FEATURES_FOLDER, "TEMP_TEST01_classes.npy"))
+def test_build_model_rfc():
+    """Test main model building method - RFC"""
+    shutil.copy(pjoin(DATA_PATH, "test_targets.npy"),
+                pjoin(cfg.FEATURES_FOLDER, "TEMP_TEST01_targets.npy"))
     shutil.copy(pjoin(DATA_PATH, "test_features.csv"),
                 pjoin(cfg.FEATURES_FOLDER, "TEMP_TEST01_features.csv"))
-    build_model.build_model("TEMP_TEST01", "TEMP_TEST01")
+    build_model.build_model("TEMP_TEST01", "RFC", "TEMP_TEST01")
     assert os.path.exists(pjoin(cfg.MODELS_FOLDER,
-                                "TEMP_TEST01_RF.pkl"))
+                                "TEMP_TEST01.pkl"))
     model = joblib.load(pjoin(cfg.MODELS_FOLDER,
-                              "TEMP_TEST01_RF.pkl"))
+                              "TEMP_TEST01.pkl"))
+    os.remove(pjoin(cfg.MODELS_FOLDER, "TEMP_TEST01.pkl"))
+    os.remove(pjoin(cfg.FEATURES_FOLDER, "TEMP_TEST01_targets.npy"))
+    os.remove(pjoin(cfg.FEATURES_FOLDER, "TEMP_TEST01_features.csv"))
     assert hasattr(model, "predict_proba")
-    os.remove(pjoin(cfg.MODELS_FOLDER, "TEMP_TEST01_RF.pkl"))
-    os.remove(pjoin(cfg.FEATURES_FOLDER, "TEMP_TEST01_classes.npy"))
+    assert "sklearn.ensemble.forest.RandomForestClassifier" in str(type(model))
+
+
+def test_build_model_rfr():
+    """Test main model building method - RFR"""
+    shutil.copy(pjoin(DATA_PATH, "test_reg_targets.npy"),
```

Comment:
I see lots of repetition of this code in all the tests.  Refactor into utility function?